documentation/modules/rn-2.6.0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+
+This release has the following technical changes:
+
+.Technical change
+
+Technical change
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Cross-Provider Features
+
+Cross-provider features. link:https://issues.redhat.com/browse/MTV-802[(MTV-802)]
+
+.RH-SDL
+
+RH-SDL. link:https://issues.redhat.com/browse/MTV-801[(MTV-801)]
+
+.OCP Provider
+
+OCP (OpenShift Container Platform) provider. link:https://issues.redhat.com/browse/MTV-739[(MTV-739)]
+
+.Console Plugin
+
+Console plugin. link:https://issues.redhat.com/browse/MTV-736[(MTV-736)]
+
+.OVA Provider
+
+OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+For 
+
+[id="resolved-issues-26_{context}"]
+== Resolved issues
```

Comment:
as commented above the following issue should move here from the known issues:
Import OVA: ConnectionTestFailed message appears when adding OVA provider

I'll add the other items once I add them to the advisory